files/angular.gravatar/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-gravatar",
+  "repo": "wallin/angular-gravatar",
+  "files": {
+    "include": ["angular-gravatar.js", "angular-gravatar.min.js"]
```

Comment:
You had to remove `build/` only from the info.ini. Here it needs to be present.
Remove `angular-gravatar.js` and leave only `build/angular-gravatar.min.js`